public/app/themes/justice/inc/search.php

```diff
@@ -19,15 +17,32 @@ public function addHooks()
         // Add a rewrite rule to handle an empty search.
         add_action('init', fn () => add_rewrite_rule('search/?$', 'index.php?s=', 'bottom'));
         // Add a rewrite rule to handle the old search urls.
-        add_action('init', [$this, 'redirectOldSearchUrls']);
+        add_action('template_redirect', [$this, 'redirectOldSearchUrls']);
+        // Add a rewrite rule to handle the search string.
         add_filter('posts_search', [$this, 'handleEmptySearch'], 10, 2);
-        add_action('pre_get_posts', [$this, 'includeDocumentsInSearchResults']);
-        add_action('pre_get_posts', [$this, 'searchFilter']);
-        add_filter('the_excerpt', [$this, 'highlightResults']);
+        // Add a query var for the parent page. This will be handled in relevanssiParentFilter.
+        add_filter('query_vars', fn ($qv) =>  array_merge($qv, array('parent')));
+
+        // Not necessary with Relevanssi.
+        // add_action('pre_get_posts', [$this, 'includeDocumentsInSearchResults']);
+        // add_action('pre_get_posts', [$this, 'searchFilter']);
+        // add_filter('the_excerpt', [$this, 'highlightResults']);
+
```

Comment:
Can the commented out code be removed if it's not needed?

Reply:
Good shout, I've removed them.